### PR TITLE
Fixes a bug when using the color mapping

### PR DESCRIPTION
**Pull request by huckocs**
Fixes a bug when using the color mapping as sprignernature-context colors.scss did not contain greyscale references.

Greyscale colors taken from references designs for Springer Nature product design language

### CODE REVIEW

File: toolkits/springernature/packages/springer-nature-context/HISTORY.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # History
 
+## 2.0.2 (2019-06-21)
```
Comment (ricky-lau) [inline toolkits/springernature/packages/springer-nature-context/HISTORY.md:3]: The date needs updating

Comment (huckocs) [reply toolkits/springernature/packages/springer-nature-context/HISTORY.md]: The dangers of copy/paste. Cheers fixed

File: toolkits/springernature/packages/springer-nature-context/scss/10-settings/colors.scss [hunk 2]
```diff
@@ -21,4 +21,15 @@ $colors: (
...
+/**
+ * Color and gray scales
+ *
+ */
+
+$page-base: #f8f;
+$greyscale-base: #555;
+$greyscale-min: 20;
+$greyscale-max: 95;
+$greyscale-stops: 5;
```
Comment (alexkilgour) [inline toolkits/springernature/packages/springer-nature-context/scss/10-settings/colors.scss:35]: where did these come from?

Comment (huckocs) [reply toolkits/springernature/packages/springer-nature-context/scss/10-settings/colors.scss:35]: The page-base and grayscale-base colors are from the sketch colour palette for Springer Nature. The min/max/stops are replicated from Springer, and need reviewing with UX. Could remove for now if prefer

**Pull request closed without merging**